cmd/ack-generate/command/controller.go

```diff
@@ -353,6 +359,144 @@ func writeDockerfile(sh *model.Helper) error {
 	return ioutil.WriteFile(path, b.Bytes(), 0666)
 }
 
+func writeConfigDirs(sh *model.Helper) error {
+	configDefaultPath := filepath.Join(optControllerOutputPath, "config", "default")
+	configControllerPath := filepath.Join(optControllerOutputPath, "config", "controller")
+	configRBACPath := filepath.Join(optControllerOutputPath, "config", "rbac")
+	if !optDryRun {
+		if _, err := ensureDir(configDefaultPath); err != nil {
+			return err
+		}
+		if _, err := ensureDir(configControllerPath); err != nil {
+			return err
+		}
+		if _, err := ensureDir(configRBACPath); err != nil {
+			return err
+		}
+	}
+	if err := writeConfigDefaultKustomizationYAML(sh); err != nil {
+		return err
+	}
+	if err := writeConfigControllerKustomizationYAML(sh); err != nil {
+		return err
+	}
+	if err := writeConfigControllerDeploymentYAML(sh); err != nil {
+		return err
+	}
+	if err := writeConfigRBACKustomizationYAML(sh); err != nil {
+		return err
+	}
+	if err := writeConfigRBACClusterRoleBindingYAML(sh); err != nil {
+		return err
+	}
+	return nil
+}
+
+func writeConfigDefaultKustomizationYAML(sh *model.Helper) error {
+	var b bytes.Buffer
+	vars := &configdefaulttemplate.ConfigDefaultKustomizationYAMLTemplateVars{
+		ServiceAlias: strings.ToLower(sh.GetServiceAlias()),
+	}
+	tpl, err := configdefaulttemplate.NewConfigDefaultKustomizationYAMLTemplate(optTemplatesDir)
+	if err != nil {
+		return err
+	}
+	if err := tpl.Execute(&b, vars); err != nil {
+		return err
+	}
+	if optDryRun {
+		fmt.Println("============================= config/default/kustomization.yaml ======================================")
+		fmt.Println(strings.TrimSpace(b.String()))
+		return nil
+	}
+	path := filepath.Join(optControllerOutputPath, "config", "default", "kustomization.yaml")
```

Comment:
As the caller method computed the desired directory that would contain the yaml and contains`ensureDir` for it, should this function take `configDefaultPath` as argument to make this statement as:
`path := filepath.Join(configDefaultPath, "kustomization.yaml")`